src/OrchardCore.Modules/OrchardCore.Workflows/Views/Activity/EditActivity.cshtml

```diff
@@ -1,7 +1,7 @@
 @model OrchardCore.Workflows.ViewModels.ActivityEditViewModel
 
 <div data-workflow-type-id="@Model.WorkflowTypeId" data-activity-id="@Model.ActivityId">
-    <h1>@RenderTitleSegments(T["Edit {0}", Model.Activity.Name.CamelFriendly()])</h1>
+    <h1>@RenderTitleSegments(T["Edit {0}", Model.Activity.DisplayText.Value.CamelFriendly()])</h1>
```

Comment:
Since you are removing the use of `CamelFriendly()` and introducing `DisplayText` instead, which I think is a good thing, you should change every single value of `DisplayText` to add spaces, like `T["User Task Event"]`.